core/utils/device/adb.py

```diff
@@ -77,23 +78,6 @@ def restart():
         Process.kill(proc_name='adb')
         Adb.run_adb_command('start-server')
 
-    @staticmethod
```

Comment:
Android Runtimes are using this method. We should change it in the tests if we delete it.

Reply:
It is now Adb.get_ids() and it is applied everywhare.